src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -554,6 +554,26 @@ describe('ReactDOMComponent', function() {
       );
     });
 
+    it('should indicate the owner when validating against invalid styles', function() {
+      var ReactTestUtils = require('ReactTestUtils');
+      expect(function() {
+        var TestComponent = React.createClass({
+          displayName: 'HelloWorld',
```

Comment:
2 things:
1. Can probably just depend on the JSX transform adding `displayName`
2. Let's move the component definition out of the expect closure and just leave the part that we actually expect to throw in.